iron-autogrow-textarea.html

```diff
@@ -341,7 +341,9 @@
       }
 
       this.bindValue = value;
-      this.$.mirror.innerHTML = this._valueForMirror();
+      this.debounce('delayMirror', function() {
+        this.$.mirror.innerHTML = this._valueForMirror();
+      }.bind(this), 500);
```

Comment:
can we use no wait at all?

Reply:
I can try without the wait, to see if it works.

Reply:
It does not work without the wait. I've obtain similar results that without the `debounce` function.

![captura de pantalla 2018-03-29 a las 9 44 20](https://user-images.githubusercontent.com/997038/38076484-0adf97b2-3336-11e8-81e0-48d933d11f8c.png)

![captura de pantalla 2018-03-29 a las 9 33 27](https://user-images.githubusercontent.com/997038/38076492-0e625546-3336-11e8-948a-950206c1c9f7.png)

I've also tested different values of waiting, but with 500 I've obtain better results and the delay on typing desappear:

![captura de pantalla 2018-03-29 a las 9 45 42](https://user-images.githubusercontent.com/997038/38076547-39aad232-3336-11e8-801a-6933794dd74b.png)

My scenery is like a chat (made with components): a user enters in a room from a list on the left, and starts typing messages. The most number of rooms you have on the left, the greater delay you feel.

In my case, the bottleneck is this line:

`this.$.mirror.innerHTML = this._valueForMirror();`

Maybe this line can be made in a different way and so the `debounce` would not be necessary.